cypress/integration/tag.js

```diff
@@ -18,7 +18,7 @@ describe('Tag page', () => {
         });
     });
     it('should contain several articles with basic information', () => {
-        cy.get('[data-test="card"]').should('have.length', 4);
+        cy.get('[data-test="card"]').should('have.length', 5);
```

Comment:
I added a new article with the rST for `tabs` since this is a new directive into the CI repo. That is why this is one more than before. I will also be using this article for creating cypress tests about the tab directive.